### PR TITLE
[libmspack] Fix several missing imports

### PR DESCRIPTION
**Pull request by lazyhamster**
- Added 2 missing export for the library.
- Removed CMAKE_DEBUG_POSTFIX which is proven to be a bad idea.
- Removed several obsolete defines in config.h.

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

File: ports/libmspack/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: libmspack
-Version: 0.10.1-2
+Version: 0.10.1-3
 Build-Depends:
```
Comment (NancyLi1013) [inline ports/libmspack/CONTROL]: There are no any ports needed to depend on. So please remove this line.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/libmspack/CONTROL (hunk 1 above)
Comment (lazyhamster) [reply ports/libmspack/CONTROL]: Fixed

**Comment by ras0219-msft** [comment]
LGTM, thanks for the PR!

**Pull request merged**